src/uint/neg.rs

```diff
@@ -3,6 +3,11 @@ use crate::{Limb, Uint, WideWord, Word, WrappingNeg};
 impl<const LIMBS: usize> Uint<LIMBS> {
     /// Perform wrapping negation.
     pub const fn wrapping_neg(&self) -> Self {
+        self.wrapping_negc().0
+    }
+
+    /// Perform wrapping negation; returns the resulting carry.
+    pub const fn wrapping_negc(&self) -> (Self, Word) {
```

Comment:
The name of it is misleading.
The operation is either wrapping or carrying (for comparison see add_wrapping/adc) hence I would rename it to just `negc`.

Reply:
done!